keycloak.yaml

```diff
@@ -62,6 +62,147 @@ subpackages:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/opt
           ln -s /usr/share/java/keycloak "${{targets.subpkgdir}}"/opt/
+  # https://github.com/bitnami/containers/tree/main/bitnami/keycloak/24/debian-12
+  - name: ${{package.name}}-bitnami-compat
```

Comment:
as it is right now (in local testing) this package contains only a single symlink

```
# apk info -L keycloak-compat
keycloak-compat-25.0.2-r0 contains:
opt/keycloak
var/lib/db/sbom/keycloak-compat-25.0.2-r0.spdx.json
```

was that intended?

Reply:
my bad. that is keycloak-compat, not bitnami-compat. ignore.